### PR TITLE
update translations in underfit-overfit.md

### PR DESCRIPTION
**Pull request by thebesttv**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-746/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-746/

File: chapter_multilayer-perceptrons/underfit-overfit.md [hunk 1]
```diff
@@ -10,58 +10,57 @@
...
 
 在接下来的章节中，我们将讨论因违背独立同分布假设而引起的问题。目前，即使认为独立同分布假设是理所当然的，理解泛化也是一个困难的问题。此外，能够解释深层神经网络泛化性能的理论基础，也仍在继续困扰着学习理论领域最伟大的学者们。
 
-当我们训练模型时，我们试图找到一个能够尽可能拟合训练数据的函数。如果该函数灵活到可以像捕捉真实模式一样容易地捕捉到干扰的模式，那么它可能执行得“太好了”，而不会产生一个对看不见的数据进行很好概括的模型。但我们想要避免这样，或者想要至少能够控制这种现象的出现。深度学习中有许多启发式的技术旨在防止过拟合。
+当我们训练模型时，我们试图找到一个能够尽可能拟合训练数据的函数。如果该函数灵活到可以像捕捉真实模式一样容易地捕捉到干扰的模式，那么它可能执行得“太好了”，而不能产生一个对看不见的数据做到很好泛化的模型。这种情况正是我们想要避免，或起码控制的。深度学习中有许多启发式的技术旨在防止过拟合。
 
 ### 模型复杂性
 
-当我们有简单的模型和大量的数据时，我们期望泛化误差与训练误差相近。当我们有更复杂的模型和更少的样本时，我们预计训练误差会下降，但泛化误差会增大。模型复杂性由什么构成是一个复杂的问题。一个模型是否能很好地泛化取决于很多因素。例如，具有更多参数的模型可能被认为更复杂。其参数有更大取值范围的模型可能更为复杂。通常，对于神经网络，我们认为需要更多训练迭代的模型比较复杂，而需要“提前停止”（early stopping）的模型（意味着具有较少训练迭代周期）就不那么复杂。
+当我们有简单的模型和大量的数据时，我们期望泛化误差与训练误差相近。当我们有更复杂的模型和更少的样本时，我们预计训练误差会下降，但泛化误差会增大。模型复杂性由什么构成是一个复杂的问题。一个模型是否能很好地泛化取决于很多因素。例如，具有更多参数的模型可能被认为更复杂。参数有更大取值范围的模型可能更为复杂。通常，对于神经网络，我们认为需要更多训练迭代的模型比较复杂，而会“提前停止”（early stopping）的模型（意味着具有较少训练迭代周期）就不那么复杂。
```
Comment (xiaotinghe) [inline chapter_multilayer-perceptrons/underfit-overfit.md:45]: “而会提前停止”意义不准确。

Comment (goldmermaid) [reply chapter_multilayer-perceptrons/underfit-overfit.md]: Fixed!

**Comment by mli** [comment]
Job d2l-zh/PR-746/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-746/

**Pull request merged**